libkbfs/test_common.go

```diff
@@ -70,11 +70,23 @@ func setTestLogger(config Config, t logger.TestLogBackend) {
 	config.SetLoggerMaker(testLoggerMaker(t))
 }
 
+// NewConfigForTest returns a ConfigLocal object suitable for testing.
+//
+// TODO: Decouple this from NewConfigLocal.
+func NewConfigForTest() *ConfigLocal {
+	config := NewConfigLocal()
```

Comment:
D'oh! Looking a bit more, looks like `NewConfigLocal` is supposed to be the 'common code' shared by everything, (since it only partially fills in the struct) so I just moved the `BlockOpsStandard` creation into `libkbfs.Init()`. WDYT?